src/ontology/phenio-edit.owl

```diff
@@ -11,49 +11,36 @@ Prefix(dcterms:=<http://purl.org/dc/terms/>)
 
 Ontology(<http://purl.obolibrary.org/obo/phenio.owl>
 
-Import(<http://purl.obolibrary.org/obo/phenio/imports/nbo_import.owl>)
```

Comment:
You hadnt updated your editfile when you switched from imports to components.

Reply:
Aha! Thanks!